src/styles/utilies/_mixins.scss

```diff
@@ -0,0 +1,14 @@
+@mixin clock-size ($size) {
+  height: $size;
+  width: $size;
+}
+
+@mixin minute-size ($height, $width) {
+  height: $height;
+  width: $width;
+}
+
+@mixin second-size ($height, $width) {
+  height: $height;
+  width: $width;
+}
```

Comment:
Here as well, two identical mixing. No need for that

Reply:
They have only one thing in common - width. So, I decided to create separate mixins. However, I will try to adhere to your advice.

Reply:
They have the same properties. The values for them are passed by variables. so you can use one mixin in two different places, just pass different values.

Reply:
> They have the same properties. The values for them are passed by variables. so you can use one mixin in two different places, just pass different values.

Thanks, I got you!:)